jcg_evaluation/src/main/scala/JSFingerprintExtractor.scala

```diff
@@ -49,16 +50,21 @@ object JSFingerprintExtractor extends FingerprintExtractor {
             for (expectedCG <- expectedCGs) {
                 val testName: String = expectedCG.filePath.split("/").last
                 if (config.debug) println("[DEBUG] Test name: " + testName + " " + algo)
-                val generatedCGFile = new File(adapterOutputDir, s"${adapter.frameworkName}/$algo/$testName")
-                val generatedCG = new AdapterCG(generatedCGFile)
-
-                // check if call graph has missing edges
-                val isSound = generatedCG.compareLinks(expectedCG).length == 0
-
-                fingerprintWriter.write(s"$testName -> $isSound\n")
+                val generatedCGFile = new File(adapterOutputDir, s"${adapter.frameworkName}/$algo/").listFiles().find(_.getName == testName).orNull
+                // if callgraph file does not exist write None to result
+                var soundSymbol = "None"
```

Comment:
There is a class hierarchy `Assesment` in jcg_evaluation that provides such symbols in a structured manner.